src/Hadolint/Lint.hs

```diff
@@ -56,10 +56,10 @@ instance Semigroup LintOptions where
       (a4 <> b4)
       (a5 <> b5)
       (a6 <> b6)
-      (min a7 b7)
+      (a7 <> b7)
```

Comment:
Interesting that in had a gut feeling about this in the previous PR that it would be better to implement monopod for the type